crates/nu-command/src/filters/default.rs

```diff
@@ -23,6 +23,11 @@ impl Command for Default {
                 SyntaxShape::String,
                 "The name of the column.",
             )
+            .switch(
+                "empty",
+                "also compact empty items like \"\", {}, and []",
```

Comment:
This description seems to be taken from #15213 

Maybe 
```suggestion
                "also overwrite empty items like \"\", {}, and []",
```

Reply:
Thanks!  Fixed in latest commit.  Used *"also replace empty items ..."*